scripts/download-binaries.sh

```diff
@@ -67,10 +67,25 @@ curl "${curl_args}O" "${golint_url}" \
     && tar xzfP "${golint_tgz}" -C "${dest_dir}" --strip-components=1 "${golint_dir}/golangci-lint" \
     && rm "${golint_tgz}"
 
+gobindata_version="3.1.2"
+gobindata_dir="go-bindata-${gobindata_version}"
+gobindata_tgz="${gobindata_dir}.tar.gz"
+gobindata_location="github.com/go-bindata/go-bindata"
+gobindata_url="https://${gobindata_location}/archive/v${gobindata_version}/${gobindata_tgz}"
+curl "${curl_args}O" "${gobindata_url}" \
+    && tar xzfP "${gobindata_tgz}" \
+    && mkdir -p ${root_dir}/vendor/${gobindata_location} \
+    && cp -R ${gobindata_dir}/* ${root_dir}/vendor/${gobindata_location} \
+    && go build -o ${dest_dir}/go-bindata ${root_dir}/vendor/${gobindata_location}/go-bindata \
+    && rm -rf ${root_dir}/vendor/${gobindata_location} \
+    && rm -rf ${gobindata_tgz} \
+    && rm -rf ${gobindata_dir}
+
 echo    "# destination:"
 echo    "#   ${dest_dir}"
 echo    "# versions:"
 echo -n "#   kubectl:        "; "${dest_dir}/kubectl" version --client --short
 echo -n "#   kubebuilder:    "; "${dest_dir}/kubebuilder" version
 echo -n "#   helm:           "; "${dest_dir}/helm" version --client --short
 echo -n "#   golangci-lint:  "; "${dest_dir}/golangci-lint" --version
+echo -n "#   go-bindata:     "; "${dest_dir}/go-bindata" -version
```

Comment:
Please ensure your editor is configured to add a newline at the end of the file.

Reply:
Fixed